RNS/Utilities/rnid.py

```diff
@@ -37,26 +37,48 @@
 ENCRYPT_EXT = "rfe"
 CHUNK_SIZE = 16*1024*1024
 
-def spin(until=None, msg=None, timeout=None):
+
+def spin(until=None, msg="", timeout=None):
     i = 0
     syms = "⢄⢂⢁⡁⡈⡐⡠"
-    if timeout != None:
+    if timeout is not None:
         timeout = time.time()+timeout
 
-    print(msg+"  ", end=" ")
-    while (timeout == None or time.time()<timeout) and not until():
+    print(msg, end="   ")
+    while (timeout is None or time.time()<timeout) and not until():
```

Comment:
this `until()` isn't actually a function, so the ()'s should be removed.  unless I'm missing something about the default value None.

Reply:
Hi!, yeah, I notice that, actually, is defined like this, with a default value of None:

![imagen](https://github.com/markqvist/Reticulum/assets/69118979/4c34d66f-8c48-41e7-b0b1-4fee288b7166)

The function spincheck declared in the middle of the code and is the value used on until()

![imagen](https://github.com/markqvist/Reticulum/assets/69118979/db846538-318f-4db9-a7ad-d2e3c5f6aa33)

Probably this needs a refactor to...

Reply:
Digging further, I see there's the `spin()` function in `rnx.py` that uses the `until` kwarg and calls to it (only in rnx.py) are using a lambda.  This may be where that pattern came from.